### PR TITLE
feat(api) - Add a guide for the rest API

### PR DESCRIPTION
**Pull request by dnkbln**
Ich habe einen Guide für die Benutzung des Wordpress Application Passwords geschrieben. 

Diese Änderung gehört zu #34

### CODE REVIEW

File: publisher/guides/api-usage.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import YouTube from 'react-youtube';
+
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password. 
+
+## Wordpress application password
```
Comment (eteubert) [inline publisher/guides/api-usage.mdx:13]:
```suggestion
## WordPress application password
```

File: publisher/guides/api-usage.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+The generation of an episode exemplifies the usage.
+
+1. Select HTTP method POST
+2. Enter the URL http://podlove.local/wp-json/podlove/v2/episodes
+3. Select the Authorization tab
+4. Select Basic Auth as type
+5. Enter username and application password
+6. Press the Send button
+7. The response (status and body) appears in the lower part of the screen
+
```
Comment (Fischaela) [inline publisher/guides/api-usage.mdx:44]: Nicely written :) Very cool guide for our API uses.  Let me know when it is live so we can tweet about it :)

**Review by Fischaela — CHANGES_REQUESTED** [review]

File: publisher/guides/api-usage.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password. 
+
+## WordPress application password
+
+This option is provided directly by WordPress (since version 5.6.). The application password is directly assigned to a user and can not use for an
+interactive login. A user can create multiple application passwords. It is recommended to assign a separate application password for each client
```
[the remark this answers is not in the document]
Comment (dnkbln) [reply publisher/guides/api-usage.mdx:16]: I try to reduce the passive sentences. I integrate your proposal.

File: publisher/guides/api-usage.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: How to use the Rest API
+---
+
+import YouTube from 'react-youtube';
+
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts to modify or query data. Here, the possibilities for authentication with the Wordpress
```
[the remark this answers is not in the document]
Comment (dnkbln) [reply publisher/guides/api-usage.mdx:10]: I integrate your feedback.

File: publisher/guides/api-usage.mdx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password. 
+
+## WordPress application password
+
+This option is provided directly by WordPress (since version 5.6.). The application password is directly assigned to a user and can not use for an
+interactive login. A user can create multiple application passwords. It is recommended to assign a separate application password for each client 
+application. If a client application compromise, the user can revoke the password specifically for this application.
```
[the remark this answers is not in the document]
Comment (dnkbln) [reply publisher/guides/api-usage.mdx:17]: I integrate your feedback.

File: publisher/guides/api-usage.mdx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+<img src="/assets/publisher/api-users.png" />
+
+WordPress admins can create Application Passwords for each user, users can also create Application Passwords for their account. Access is through 
+the user profile page.
+
+<img src="/assets/publisher/api-application-password-create.png" />
+
+It is important to save the now visible password securely right away. After leaving the page, the password is no longer visible and 
+cannot be displayed again. In the list all created Application Passwords are listed and can revoke individually.
+
```
[the remark this answers is not in the document]
Comment (dnkbln) [reply publisher/guides/api-usage.mdx:30]: I integrate your feedback.

File: publisher/api.json [hunk 7]
```diff
@@ -3894,7 +3894,7 @@
           "soundbite_start": "00:00:10",
           "soundbite_duration": "00:00:59",
           "soundbite_title": "A short soundbite from the episode",
-          "explizit": "no"
+          "explicit": "no"
```
Comment (alexander-heimbuch) [inline publisher/api.json:3897]: sollte glaube ich `false` sein

File: publisher/guides/api-usage.mdx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+title: How to use the Rest API
+---
+
+import YouTube from 'react-youtube';
+
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts in order to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password.
```
Comment (alexander-heimbuch) [inline publisher/guides/api-usage.mdx:11]: Als Vorschlag: "To authenticate the requests against the Podlove Publisher API we recommend the WordPress application password functionality. In the example you can also see how you can test the API with Postman and the WordPress application passwords."

File: publisher/guides/api-usage.mdx [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+# How to use the Rest API with authentification
+
+All changes to data using the API require the authentication of the user. In addition, only authenticated users can query non-public data. 
+An authorized user must have the right to modify posts in order to modify or query data. Here, the possibilities for authentication with the Wordpress 
+Application Password are presented. Furthermore, it is explained how the API can be tested with Postman and Application Password. 
+
+## WordPress application password
+
+This option is provided directly by WordPress (since version 5.6.). The application password is directly assigned to a user and can not be used for an
+interactive login. A user can create multiple application passwords. It is recommended to assign a separate application password for each client
```
Comment (alexander-heimbuch) [inline publisher/guides/api-usage.mdx:16]: A user can create multiple application passwords and it is recommended to assign a separate application password for each client application for security reasons.

File: publisher/guides/api-usage.mdx [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+This option is provided directly by WordPress (since version 5.6.). The application password is directly assigned to a user and can not be used for an
+interactive login. A user can create multiple application passwords. It is recommended to assign a separate application password for each client 
+application. If a client application was compromised, the user can revoke the password specifically for this application.
+
+### Create an application password
+
+<img src="/assets/publisher/api-users.png" />
+
+WordPress admins can create Application Passwords for each user, users can also create Application Passwords for their account. Access is through 
+the user profile page.
```
Comment (alexander-heimbuch) [inline publisher/guides/api-usage.mdx:24]: WordPress admins can create Application Passwords for each user, users can also create Application Passwords for their account in the users profile page.

**Review by alexander-heimbuch — APPROVED** [review]

File: publisher/guides/api-usage.mdx (hunk 8 above)
Comment (dnkbln) [reply publisher/guides/api-usage.mdx:11]: Dein Vorschlag ist als Einleitung gemeint, richtig?

File: publisher/guides/api-usage.mdx (hunk 9 above)
Comment (dnkbln) [reply publisher/guides/api-usage.mdx:16]: Habe ich geändert.

File: publisher/guides/api-usage.mdx (hunk 10 above)
Comment (dnkbln) [reply publisher/guides/api-usage.mdx:24]: Habe ich geändert

**Pull request merged**